server/sonar-server/src/main/java/org/sonar/server/component/ws/SearchProjectsAction.java

```diff
@@ -323,6 +324,15 @@ private SearchProjectsWsResponse buildResponse(SearchProjectsRequest request, Se
           .forEach(response::addComponents);
         return response;
       })
+      .map(response -> {
```

Comment:
I have the feeling it should optional, at least I don't feel comfortable to add organization with a more global perspective